src/utils/cache.rs

```diff
@@ -111,9 +111,8 @@ pub trait CacheExt: Cache {
             Some(cached) => CachedWithCode::split(&cached).to_result(),
             None => {
                 let mut request = client.get(url);
-                if request_timeout > 0 {
-                    request = request.timeout(Duration::from_millis(request_timeout));
-                }
+                request = request.timeout(Duration::from_millis(request_timeout));
```

Comment:
removed the check against `0` as there is a default now

Reply:
What happens if we pass in 0 now?

Reply:
you can pass 0 and it will immediately timeout. I think that you used 0 in the fix for the safe app info to simply not set a request timeout. In this case, we are always setting it to `request_timeout` which should be the `DEFAULT_REQUEST_TIMEOUT = 5000`